src/Microsoft.ML.Data/Transforms/KeyToVector.cs

```diff
@@ -246,14 +246,24 @@ public Mapper(KeyToVectorMappingTransformer parent, DataViewSchema inputSchema)
                 _parent = parent;
                 _infos = CreateInfos(inputSchema);
                 _types = new VectorDataViewType[_parent.ColumnPairs.Length];
-                for (int i = 0; i < _parent.ColumnPairs.Length; i++)
+                try
```

Comment:
Please add a note here as to why this change is being introduced.

Reply:
sure